tests/unit/-private/properties/triggerable-test.js

```diff
@@ -43,6 +40,44 @@ moduleForProperty('triggerable', function(test) {
     return this.adapter.wait();
   });
 
+  test("calls Ember's triggerEvent helper with dynamic event options", function(assert) {
+    assert.expect(1);
+
+    let page = create({
+      foo: triggerable('keypress', 'span')
+    });
+
+    this.adapter.createTemplate(this, page, '<span></span>');
+
+    this.adapter.triggerEvent((actualSelector, _, event, options) => {
+      assert.equal(options.keyCode, 13);
+    });
+
+    page.foo({ keyCode: 13 });
+
+    return this.adapter.wait();
+  });
+
+  test("overrides static event options with dynamic event options", function(assert) {
```

Comment:
I think the description is clear. I used to use the `at definition time` and `at execution time` to differentiate when defining the descriptor or executing the generated function. But I like the dynamic/static idea.